src/main/groovy/wooga/gradle/unity/UnityPluginConventions.groovy

```diff
@@ -52,6 +52,12 @@ class UnityPluginConventions implements PlatformUtilsImpl {
      */
     static File UNITY_LICENSE_DIRECTORY_MAC_OS = new File("/Library/Application Support/Unity/")
 
+    /**
+     * {@code File} to Unity license directory on macOS.
+     * @value "/Library/Application Support/Unity/"
+     */
+    static File UNITY_LICENSE_DIRECTORY_LINUX = new File(".local/share/unity3d/Unity/")
```

Comment:
We have to be carefull here. We should use the `$HOME` variable or the system property `user.home` to create a absolute path based on the user home.

Reply:
So it would be $HOME/shared/... ?

Reply:
no. Either `"${System.env("HOME")}/.local/....` or `"${System.getProperty("user.home")}/.local/...`

Reply:
SO says the property way is the best
https://stackoverflow.com/a/586345/1380824

Reply:
Thank you.